### ISSUE TITLE
using removed celery functionality

### ISSUE THREAD

**Issue by JensTimmerman** [nikmolnar/django-task-api#20]
updating to latest celery makes this project fail

```
  File "/home/jens/.local/lib/python3.9/site-packages/task_api/apps.py", line 8, in ready
    from .backends import celery  # Make sure task is registered
  File "/home/jens/.local/lib/python3.9/site-packages/task_api/backends/celery.py", line 5, in <module>
    from celery import task
  File "/home/jens/.local/lib/python3.9/site-packages/celery/local.py", line 497, in __getattr__
    module = __import__(self._direct[name], None, None, [name])
ModuleNotFoundError: No module named 'celery.task'
```

**Comment by nikmolnar** [comment]
Hi @JensTimmerman , thanks for reporting this issue! I'll look into it and see if I can get it working with the latest Celery release.

**Comment by nikmolnar** [comment]
This is fixed in version 1.3.0.
